go-controller/pkg/ovn/pods.go

```diff
@@ -374,29 +376,40 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
 			return fmt.Errorf("unable to create LSPSetDynamicAddresses command for port: %s", portName)
 		}
 		cmds = append(cmds, cmd)
-	} else {
+
+		// ensure we have/can allocate the IP in the annotation
```

Comment:
shouldn't this comment read "ensure we can allocate the IP on the port" ?

Reply:
in fact it should say ensure we have an IP allocated based on what's in the annotation.

Reply:
yeah will correct this to say "ensure we have reserved the IPs in the annotation"